src/olympia/amo/cron.py

```diff
@@ -159,28 +159,6 @@ def collection_subscribers():
 
 
 @cronjobs.register
-def unconfirmed():
```

Comment:
@EnTeQuAk I'm not sure of the situation re: unconfirmed user deletion in the past. Definitely seems odd that a crontab there isn't being run as expected - it would be good to understand what's going on there.